crates/types/src/data.rs

```diff
@@ -947,6 +947,7 @@ impl<TYPES: NodeType> Committable for SequencingLeaf<TYPES> {
     fn commit(&self) -> commit::Commitment<Self> {
         // Commit the block commitment, rather than the block, so that the replicas can reconstruct
         // the leaf.
+        // TODO GG why can't we use block.commit() here?
```

Comment:
What I mean is: why can't we simply return `block_commitment` instead of re-hashing the commitment along with all this other metadata? Why can't the commitment of a `SequencingLeaf` simply be equal to the block commitment?

Reply:
We discussed during a meeting. The purpose of SequencingLeaf commitment is so that nodes can sign specific metadata about the block's position in the chain, etc. It's not enough to just sign the payload commitment.